KH3Randomizer/Components/OptionComponents/WorldTreasureSubComponent.razor

```diff
@@ -9,7 +9,13 @@
         <th>Action</th>
     </TableHeader>
     <RowTemplate>
-        <td>@Treasure.Id.KeyIdToDisplay()</td>
+        <td>
+            @Treasure.Id.KeyIdToDisplay()
+            <Tooltip Text="@Treasure.Id.KeyIdToDisplay().GetChestLocation(GetDataTableEnumFromSelection(this.WorldName))">
+                <i class="bi bi-question-circle" style="font-size: 1rem; color: #0366de;"></i>
```

Comment:
Love that this was gotten around to, I've just been too busy to implement Clara_The_Classy's treasure locations! Thank you!